src/sentry/data_export/tasks.py

```diff
@@ -66,7 +69,7 @@ def assemble_download(data_export_id, limit=None, environment_id=None):
     except NotImplementedError as error:
```

Comment:
Should we be adding `data.error` metrics for these cases too? My sense is that we'll want to know when an `ExportError` or `NotImplementedError` occurs.

Reply:
That is really only there for someone sending discover queries, that bit will be removed once the discover PR is merged